core/vm/opcodes.go

```diff
@@ -25,7 +25,7 @@ type OpCode byte
 
 // IsPush specifies if an opcode is a PUSH opcode.
 func (op OpCode) IsPush() bool {
-	return PUSH1 <= op && op <= PUSH32
+	return PUSH0 <= op && op <= PUSH32
```

Comment:
I believe there is one PR missed from go-ethereum: https://github.com/ethereum/go-ethereum/commit/6f929a0762be92130588779a8535ed0e3fc58d87. 
Even the upstream op-get missed it.  
And it may cause one issue in disassembler when processing PUSH0, because the method IsPush() is changed here